CoreScriptsRoot/ServerCoreScripts/ClientChat/ChatSettings.lua

```diff
@@ -0,0 +1,44 @@
+local source = [[
+-- This module is for initial settings that are not intended to be changed during runtime.
+
+local module = {}
+
+module.WindowDraggable = false
+module.WindowResizable = false -- soon (tee emm)
```

Comment:
It's not a feature that absolutely needs to be completed for the chat system to function as well as the old one. The setting is just there as a placeholder for now. It's like #1 on the new features to-do list for afterwards.